packages/html/src/StyledTable.ts

```diff
@@ -0,0 +1,53 @@
+import { SimpleTable } from "./SimpleTable";
+
+export class StyledTable extends SimpleTable {
+    constructor() {
+        super();
+    }
+    update(domNode, element) {
+        super.update(domNode, element);
+
+        this.theadColumnStyles().forEach((styleObj, i) => {
+            this.applyStyleObject(element.select(`.th-${i}`), styleObj);
+        });
+        this.tbodyColumnStyles().forEach((styleObj, i) => {
+            this.applyStyleObject(element.selectAll(`.col-${i}`), styleObj);
+        });
+        if (this.evenRowStyles_exists()) {
+            this.applyStyleObject(element.selectAll("tbody > tr:nth-child(even)"), this.evenRowStyles());
```

Comment:
This will break when the data changes - specifically if a single row is removed at the top?